src/helpers/erc20.ts

```diff
@@ -14,13 +14,12 @@ import { inputValueToHex } from '@utils';
 interface IFormatApproveTxInputs {
   contractAddress: ITxToAddress;
   baseTokenAmount: Bigish;
-  fromAddress: ITxFromAddress;
+  fromAddress?: ITxFromAddress;
```

Comment:
When is this optional?

Reply:
When we are creating an approval tx for swap but don't know the sender yet fx.

Reply:
very well